src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -117,6 +117,7 @@ public void moveModifications(UUID groupUuid, List<UUID> modifications, UUID bef
         newModificationList.addAll(index, modificationsToMove);
 
         modificationGroupEntity.setModifications(newModificationList);
+        return List.of();
```

Comment:
I think it's better to retuen `void` for a better understanding of the code
Conceptually the `move` returns nothing

Reply:
yes